### DISCUSSION TITLE
iOS Widget doesn't work if I deploy my React Native app using CodeMagic.

### DISCUSSION THREAD

**Discussion by vovaparkhomchuk** [codemagic-ci-cd/codemagic-docs#1454] [Q&A] [answered]
So, I have a widget inside my app. It operates the data using Apple App Groups. 

There are some missing entitlements in the build made by codemagic. 

When I deploy my app just using Xcode everything works fine. 

Here is my build id: 64025d03bd5472e173d67e89. Could you help me, please?

**Comment by himesh-cm** [comment, marked as answer]
Let me check, i will get back to you

**Reply by himesh-cm** [reply]
Hi @vovaparkhomchuk, can you please remove the below flag and try re-running the build?

```
--archive-xcargs "CODE_SIGNING_REQUIRED=YES CODE_SIGNING_ALLOWED=NO"
```

**Reply by vovaparkhomchuk** [reply]
It's worked!!! Thank you so much!!!
